store/index/index.go

```diff
@@ -90,119 +92,226 @@ const BucketPoolSize = 1024
 //
 // It is created if there is no existing index at that path.
 func OpenIndex(path string, primary primary.PrimaryStorage, indexSizeBits uint8) (*Index, error) {
+	var lastIndexNum uint32
+	var lastIndexPath string
 	var file *os.File
```

Comment:
Being picky: I find moving these closer to the block in which they are first used makes the code easier to read.

Reply:
I like picky.  Done.